sw/device/tests/BUILD

```diff
@@ -4398,8 +4398,10 @@ test_suite(
         cw310 = cw310_params(
             needs_jtag = True,
             otp = _SIVAL_OTP_IMAGE[lc_state],
+            # TODO(#22823): Remove "broken" tag once the test is fixed.
             tags = [
                 "lc_{}".format(lc_state),
+                "broken",
```

Comment:
I think the `broken` tag should not be used when the test is not broken but the bitstream is.

Perhaps the `skip_in_ci` tag is better to skip these tests in CI.

Reply:
Hm, I think I would be prefer broken here, just because when we grep for "skip_in_ci" these are tests that should still work locally and are often (or should ideally be) run in a regular cadence regression, but are not run in pre-submit intentionally, usually due to resource constraints. To me, "Broken" indicates it is actually broken locally too. In this case, given our bitstream caching mechanism, this test is indeed broken locally too. Whether it is a software or hardware issue that is the cause of the brokenness is not as important IMO.